### ISSUE TITLE
A11y_Azure data studio_Extentions_Query History_Keyboard: When we tab to "Disable button" after pressing on right arrow key the keyboard focus is on Disable button itself instead of moving to Uninstall button.

### ISSUE THREAD

**Issue by Jtith** [microsoft/azuredatastudio#20877]
### GitHub Tags:
#A11yMAS; #A11yTCS; #A11ySev2; #SH_SQLAzureDataStudio_Win32_Oct2022; #SQL Azure Data Studio; #WCAG2.1.1; #Keyboard; #Win11; #FTP; #DesktopApp; #Win32; #NonBenchmark;

### Environment details:
Application Name: SQL Azure Data Studio
Windows 11 22H2 Version OS Build 22621.674

### Repro Steps:

1. Install Azure Data Studio – Insiders
2. Tab till New connection and press enter.
3. Using valid credentials, connect to server.
4. Tab till extensions and press enter.
5. Tab till search box and search "Query History". 
6. Tab till disable button
7. Press on right arrow key to check whether keyboard focus is taking twice to reach install/uninstall button or not.
### Actual Result:
When we tab to "Disable button" after pressing on right arrow key the keyboard focus is on Disable button itself instead of moving to "Uninstall button.

### Expected Result:
When we tab to "Disable button" after pressing on right arrow key the keyboard focus should go to Uninstall button.

### User Impact:
Keyboard user will get confused if the focus takes two times to receive focus on uninstall button.

Attachment:
![A11y_Azure data studio_Extentions_Query History_Keyboard](https://user-images.githubusercontent.com/100552367/196693812-10fb3e2b-11c4-4b19-bff4-0c6101c2df64.gif)

**Comment by Charles-Gagnon** [comment]
Upstream issue, filed an issue for it in VS Code : https://github.com/microsoft/vscode/issues/164050

@Jtith - can we close this one? If they fix it there we'll port the fix over

**Comment by Jtith** [comment]
@Charles-Gagnon Once this https://github.com/microsoft/vscode/issues/164050 bug fixed and closed. we will verify in our environment and then we can close this bug. Please don't close this bug directly it should be closed by accessibility tester only. Thanks!

**Comment by Ashutoshsahoo164** [comment]
Hi @Jtith and @Charles-Gagnon  Could you please verify and close this bug? Thanks

**Comment by Jtith** [comment]
@Charles-Gagnon Verified in version 1.40.0 issue is still repro. Could you please provide us latest build version to verify and close the bug

**Comment by Charles-Gagnon** [comment]
You need to install the https://github.com/Microsoft/azuredatastudio#try-out-the-latest-insiders-build-from-main and use that. This should be showing as version 1.41.0 currently.

And in general - all accessibility testing should be done with the latest insiders build and you should be updating it to the latest (daily) build every time you start testing.

**Comment by Jtith** [comment]
verified in the latest build 1.41.0 issue is not repro [20877_Issue is fixed.zip](https://github.com/microsoft/azuredatastudio/files/10302265/20877_Issue.is.fixed.zip) hence closing the bug

**Comment by InduPriya1805** [comment]
GitHubTags:#SH-SQL Azure Data Studio-Win32-OCT2022
